RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -976,13 +987,10 @@ namespace mkfit {
 
             if (m_iteration_hit_mask && (*m_iteration_hit_mask)[hi_orig]) {
               dprintf(
-                  "Yay, denying masked hit on layer %u, hi %u, orig idx %u\n", L.layer_info()->layer_id(), hi, hi_orig);
+                  "Yay, denying masked hit on layer %u, hi %u, orig idx %u\n", L.layer_info().layer_id(), hi, hi_orig);
               continue;
             }
 
-            if (m_XHitSize[itrack] >= MPlexHitIdxMax)
-              break;
```

Comment:
how significant is this change? it seems like the only change that can affect phase-1 setup